Add own Github username to contributors.json

rjepstein opened the pull request:
#### 077b5c0278963974584b1e99936932cbd88a90e6
<pre>
Add own Github username to contributors.json

Reviewed by Alexey Proskuryakov.

* metadata/contributors.json: Add Github username.

Canonical link: <a href="https://commits.webkit.org/252612@main">https://commits.webkit.org/252612@main</a>
</pre>

webkit-commit-queue commented:
@rjepstein does not have committer permissions according to https://raw.githubusercontent.com/WebKit/WebKit/main/metadata/contributors.json.

If you do have committer permmissions, please ensure that your GitHub username is added to contributors.json.

Rejecting 8dfc2a11d88c88ab8d303a1dce3fb901c1561f29 from merge queue.

rjepstein committed:
Add own Github username to contributors.json

Reviewed by Alexey Proskuryakov.

* metadata/contributors.json: Add Github username.

Canonical link: https://commits.webkit.org/252612@main

webkit-commit-queue commented:
Committed 252612@main (077b5c027896): <https://commits.webkit.org/252612@main>

Reviewed commits have been landed. Closing PR #2554 and removing active labels.
